### PR TITLE
Additional Orchard Integration work

### PR DESCRIPTION
**Pull request by spamhurts**
#### Rationale
Orchard HL7 Integration

#### Related Pull Requests
None

#### Changes
-Added remarks to clinpathRunsAll so they can be displayed in the clinical history
-Fixed issue sorting test results within panels
-Grouped all labwork results into the Labwork category 
-Added JavaScript trigger for taqmanResults to insert objectId per row 
-Fixed Urinalysis TableTitle in metadata
-Removed unused table from datasets_manifest.xml
-Removed ObjectId from TaqmanResults configuration in favor of shared configuration 
-Code cleanup in p_load_hl7_data.sql
-updated .gitignore to ignore generated credits

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -32,3 +32,5 @@ snprc_ehr/.editorconfig
 snprc_ehr/resources/referenceStudy/LoadTaqmanData
 snprc_ehr/resources/referenceStudy/datasetImport/LoadTaqmanData
 snprc_scheduler/.gradle
+snprc_ehr/resources/credits
```
Comment (jallentxbiomed) [inline .gitignore:35]: What file is this? Generated locally?

Comment (spamhurts) [reply .gitignore:35]: This is the directory that has the dependencies.txt file generated in it - I think it's getting generated when the apache-commons-text lib gets built.

File: snprc_ehr/src/org/labkey/snprc_ehr/history/LabworkDataSource.java [hunk 2]
```diff
@@ -190,12 +191,7 @@ protected String getPrimaryGroup(Results rs) throws SQLException
...
     {
         Set<String> types = new HashSet<>();
-        types.add("Labwork");
-        for (LabworkType type : EHRService.get().getLabworkTypes(c))
-        {
-            types.add(type.getName());
-        }
-
-        return types;
+        types.add(_labwork_category);
```
Comment (jallentxbiomed) [inline snprc_ehr/src/org/labkey/snprc_ehr/history/LabworkDataSource.java:194]: Why is a hash set being returned if it is just being populated with one string?

Comment (spamhurts) [reply snprc_ehr/src/org/labkey/snprc_ehr/history/LabworkDataSource.java:194]: ClinicalHistoryManager.java in the EHR module expects the category types to be in a hash set. I decided to group all the labwork types into a single category for display purposes on the clinical history page to make it easier to enable/disable labwork results.

**Review by jallentxbiomed — APPROVED** [review]

**Pull request merged**